src/tests/ftest/rebuild/cascading_failures.py

```diff
@@ -76,6 +76,9 @@ def execute_during_rebuild(self):
             self.server_managers[0].stop_ranks(
                 [self.inputs.rank.value[1]], self.d_log)
 
+        #Wait for rebuild completion
+        self.pool.wait_for_rebuild(False, 1)
```

Comment:
probably wait a few seconds after stop ranks to make sure rebuild really happened?

Reply:
agreed, this patch probably can be closed and we can wait test engineer's refine.